engine/src/main/java/org/terasology/config/facade/BindsConfiguration.java

```diff
@@ -10,24 +10,54 @@
 import java.util.List;
 
 /**
- * Facade for {@link Config#getBinds()}
+ * Facade for {@link Config#getBinds()}.
+ * The binds configuration holds the mapping from bind uris.
```

Comment:
...the mapping from bind(ing?) uris.... to what? It feels like the sentence is cut short.